pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -244,6 +248,10 @@ func (lrw *logicalReplicationWriterProcessor) Start(ctx context.Context) {
 		}
 		return nil
 	})
+
+	if err := lrw.dlqClient.Create(); err != nil {
+		lrw.sendError(errors.Wrap(err, "create dead letter queue client"))
```

Comment:
nit: errors usually are are written in past tense. "failed to create dead letter queue" would be more appropriate